src/features/currentTodo.ts

```diff
@@ -28,7 +24,11 @@ const currentTodoReducer = (
```

Comment:
Comments should be placed on a separate line above the code they are describing. Add a comment to describe the purpose of the 'currentTodoReducer' function.